tests/Tests/Arithmetization.hs

```diff
@@ -20,8 +20,8 @@ import           ZkFold.Symbolic.Types                       (Symbolic)
 propCircuitInvariance :: Arithmetic a => (ArithmeticCircuit 1 a, a, a) -> Bool
 propCircuitInvariance (ac, x, y) =
     let ac' = mapVarArithmeticCircuit ac
-        v   = ac `eval` fromList [(1, x), (2, y)]
-        v'  = ac' `eval` fromList [(1, x), (2, y)]
+        v   = ac `eval` fromList [(0, x), (1, y)]
+        v'  = ac' `eval` fromList [(0, x), (1, y)]
```

Comment:
Generalize this code according to the new `arbitrary` instance for `ArithmeticCircuit`.

Reply:
Use `ArithmeticCircuitTest` as input.